docs/content/how-to/mce/Dual/hostedcluster/worker-nodes.md

```diff
@@ -0,0 +1,59 @@
+Regarding the worker nodes, if you are working on real bare metal, this step is to make sure that the details set in the `BareMetalHost` are well set, if not you will need to debug why does not work properly.
+
+If you are working with virtual machines, we can follow this steps and create empty ones to be consumed by Metal3 operator.
+
+For this purpose we will use again Kcli.
+
+## Creating Virtual Machines
+
+If this is not your first try, you will need first to delete the prior try, to do so, please go to the [Deleting Virtual Machines](#deleting-virtual-vachines) section.
```

Comment:
```suggestion
If this is not your first try, you will need to first delete the prior try. To do so, please go to the [Deleting Virtual Machines](#deleting-virtual-vachines) section.
```

Reply:
I've modified the style a bit, I think is better now.